server/locale/FR/mutations/gpl.js

```diff
@@ -86,5 +86,53 @@ export default {
             })
         ],
         ...textLogoMutations
+    ],
+    'layout:flex': [
+        [
+            'default',
+            {
+                logo: Logo.PP_PAYPAL.WHITE,
+                headline: [
+                    {
+                        tag: 'xsmall'
+                    },
+                    {
+                        tag: 'medium'
+                    }
+                ],
+                disclaimer: ['default']
+            }
+        ],
+        [
+            'ratio:1x1',
+            {
+                subHeadline: [{ tag: 'small', br: ['achats'] }],
+                styles: [
+                    `.message__sub-headline { font-size: 1.2rem; } .message__disclaimer { bottom: 12%; }`,
+                    `@media screen and (max-width: 240px) { .message__sub-headline { font-size: 0.9rem; } }`
```

Comment:
I think we can remove the styles array from this file. I'll add a comment down below to show some small tweaks and where I would add them back.